src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/Signals.java

```diff
@@ -18,10 +18,14 @@
 final class Signals
 {
     public static final int CACHE_ENTRY_UPDATED_SIGNAL = 1;
-    public static final int CACHE_ENTRY_SIGNAL = 2;
     public static final int CACHE_ENTRY_ABORTED_SIGNAL = 3;
     public static final int REQUEST_EXPIRED_SIGNAL = 4;
-    public static final int INITIATE_REQUEST_SIGNAL = 5;
+    public static final int REQUEST_GROUP_LEADER_UPDATED_SIGNAL = 5;
+    public static final int GROUP_REQUEST_RETRY_SIGNAL = 7;
+    public static final int CACHE_ENTRY_NOT_MODIFIED_SIGNAL = 8;
+    public static final int CACHE_ENTRY_READY_SIGNAL = 9;
+    public static final int REQUEST_ABORTED_SIGNAL = 10;
```

Comment:
Where is this last signal still used?

Reply:
it is used in `HttpCacheProxyCachedRequest ` to signal that the request got aborted while it is service the response from cache.